acme_corp/do_task.py

```diff
@@ -4,22 +4,20 @@
 
 logger = logging.getLogger(__name__)
 
+# using Sphinx docstrings style: <https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html>
```

Comment:
did we decide to switch to numpy docstring style?

Reply:
I think we decided to make that a separate PR #12.